docs/examples/1.6.x/client-android/java/functions/get-deployment-download.md

```diff
@@ -0,0 +1,23 @@
+import io.appwrite.Client;
+import io.appwrite.coroutines.CoroutineCallback;
+import io.appwrite.services.Functions;
+
+Client client = new Client(context)
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("&lt;YOUR_PROJECT_ID&gt;"); // Your project ID
```

Comment:
We need to figure out why this is html entities but the params like below aren't 😕

Reply:
We've got a separate PR to fix it 👍